api/routers/areas.py

```diff
@@ -230,8 +293,10 @@ def delete_area_occupancy_rules(area_id: str):
         with open(area_config_path, "r") as area_file:
             data = json.load(area_file)
     else:
+        # Doubt: I believe the line below does not have sense at all. why are we sending a 204 status code?
```

Comment:
This is copied from how other DELETE work but yeah, maybe there is a better status code for this

Reply:
I mean, the status code here doesn't matter because since the second parameter is False, the handle_response function will always throw a 500 error. I'll remove the 204 status code to avoid confusion.